xapian-core/queryparser/termgenerator_internal.h

```diff
@@ -43,7 +44,7 @@ class TermGenerator::Internal : public Xapian::Internal::intrusive_base {
     WritableDatabase db;
 
   public:
-    Internal() : strategy(STEM_SOME), stopper(NULL), termpos(0),
+    Internal() : strategy(STEM_SOME), stop_mode(STOPWORDS_NONE), stopper(NULL), termpos(0),
```

Comment:
Is this correct? If I create a `TermGenerator` and set its stopper, do we not want the default behaviour to be `INDEX_UNSTEMMED_ONLY`, as before?